src/app/admin/_components/event/CreateEvent.js

```diff
@@ -0,0 +1,79 @@
+import { Create, TextInput, SimpleForm, DateTimeInput, SelectInput, NumberInput, required } from 'react-admin';
+import { useWatch } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useState } from 'react';
+import { CreateEventSchema } from '@admin/_lib/validationSchemas/createEventSchema';
+import { transformEventData } from '@admin/_utils/transformEventData';
+import { TagSelect } from './TagSelect';
+
+const fieldGroupClass = 'flex flex-col md:flex-row md:gap-6';
+
+function PriceInput() {
+  const priceType = useWatch({ name: 'priceType' });
+  return (
+    <NumberInput disabled={priceType !== 'FIXED_PRICE' && priceType !== 'MIN_PRICE'} label="Ціна" source="price" />
```

Comment:
🟢 you can simplify by checking `priceType === 'FREE'` i guess 🤔

Reply:
changed it